src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -108,20 +108,24 @@ final class OnboardingInfoViewController: UIViewController {
 		if pageType == .togetherAgainstCoronaPage { loadCountryList() }
 	}
 
+	
```

Comment:
Is 'double empty line' the new style for the code?

Reply:
Do we have a rule for this? I prefer two empty lines between funcs.

Reply:
Keep it as it is! :) We can discuss solutions here in the next Dev Camp.

Reply:
We only have on line between funcs basically everywhere else, so it would be better for the sake of consistency to only have one in this file as well